ui/packages/app/src/components/utils/Webcam.svelte

```diff
@@ -66,10 +66,3 @@
 		/>
 	</button>
 </div>
-
-<style lang="postcss" global>
-	video {
-		-webkit-transform: scaleX(-1);
-		transform: scaleX(-1);
-	}
-</style>
```

Comment:
I thought we needed these lines because the webcam was flipped (#627)?

Reply:
added back, but removed global, as per @pngwn suggestion